### PR TITLE
feat: add help for company number and reference number

### PR DESCRIPTION
**Pull request by n-r-b**
### JIRA link
https://companieshouse.atlassian.net/browse/LFA-1263

### Change description
Help for company number and reference number on Penalty Details Entry page.

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes look good on mobile
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/views/penalty-details.njk [hunk 1]
```diff
@@ -54,6 +55,21 @@
             autocomplete: "off"
           }) 
         }}
+        <div class='govuk-form-group'>
+          {% set detailsHTML %}
+          <p>
+            Your company number and reference number are shown at the bottom of your penalty notice.
+          </p>
+          <img src='{{ ROOT_URI }}/images/penalty-notice-image.png' class='call-out-image' alt=''/>
+          {% endset -%}
```
Comment (akaJSaunders) [inline src/views/penalty-details.njk]: HTML components should remain as single quoutes

Comment (n-r-b) [reply src/views/penalty-details.njk]: double.. 👍

File: src/views/penalty-details.njk [hunk 2]
```diff
@@ -54,6 +55,21 @@
             autocomplete: "off"
           }) 
         }}
+        <div class="govuk-form-group">
+          {% set detailsHTML %}
+          <p>
+            Your company number and reference number are shown at the bottom of your penalty notice.
+          </p>
+          <img src="{{ ROOT_URI }}/images/penalty-notice-image.png" class="call-out-image" alt=""/>
```
Comment (geek-soft) [inline src/views/penalty-details.njk]: Being picky a bit here but I do not see much value in keeping word image in the image name :-)

Comment (n-r-b) [reply src/views/penalty-details.njk]: i've changed this

**Pull request merged**